### PR TITLE
Fix m_pa* variable naming to m_ap*

### PR DESCRIPTION
**Pull request by ChillerDragon**
Fixed using the following shell script

```
#!/bin/sh

grep -rEo '\bm_paa[A-Z][a-zA-Z0-9]+' src | cut -d':' -f2- | sort -u |
while read -r var
        do
                new="m_aap$(printf '%s' "$var" | cut -c 6-)"
                perl -i -p -e's/\b'"$var"'\b/'"$new"'/g' $(find . -name
'*.h' -o -name '*.cpp' -o -name '*.c')
        done

grep -rEo '\bm_pa[A-Z][a-zA-Z0-9]+' src | cut -d':' -f2- | sort -u |
while read -r var
        do
                new="m_ap$(printf '%s' "$var" | cut -c 5-)"
                perl -i -p -e's/\b'"$var"'\b/'"$new"'/g' $(find . -name
'*.h' -o -name '*.cpp' -o -name '*.c')
        done
```

### CODE REVIEW

File: src/engine/client/textrender.h [hunk 1]
```diff
@@ -162,7 +162,7 @@ class CTextRender : public IEngineTextRender
 	// support regional variant fonts
 	int m_NumVariants;
 	int m_CurrentVariant;
-	CFontLanguageVariant *m_paVariants;
+	CFontLanguageVariant *m_apVariants;
```
Comment (heinrich5991) [inline src/engine/client/textrender.h:165]: `m_paVariants` or `m_pVariants` seems to be correct here.

Comment (oy) [reply src/engine/client/textrender.h:165]: Yeah, @ChillerDragon could you remove the change to that variable?

Comment (ChillerDragon) [reply src/engine/client/textrender.h]: I went with `m_pVariants` since that seems the most correct. I don't understand where the `a` is coming from.

Comment (oy) [reply src/engine/client/textrender.h]: 👍

**Pull request merged**